CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# [0.0.1]
```

Comment:
Let's start with 1.0.0
pls change it to all the different places you've mentioned 0.0.1

Reply:
version changed.